en/docs/learn/examples/protocol-switching/switching-from-udp-to-https.md

```diff
@@ -13,11 +13,11 @@ Following are the integration artifacts (proxy service) that we can used to impl
         <inSequence>
             <log level="full"/>
             <property name="OUT_ONLY" value="true"/>
-            <send>
+            <call>
                 <endpoint>
                     <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
                 </endpoint>
-            </send>
+            </call>
```

Comment:
Shouldn't we need a respond mediator here.

Reply:
Since the `OUT_ONLY` property has been set to `true`, we do not expect a response.